irohad/torii/command_client.hpp

```diff
@@ -52,6 +52,8 @@ namespace torii {
     std::unique_ptr<iroha::protocol::CommandService::Stub> stub_;
     grpc::CompletionQueue completionQueue_;
     grpc::Status status_;
+
+    enum class State { RequestCreated, ResponseSent };
```

Comment:
What is the purpose of `State`?

Reply:
This enum was initially somewhere deep inside of grpc_call of something like that and it was used not only by a server side but also a client one so I wasn't able to remove it without reworking a client too. But I am pretty sure it could be deleted after your pr about removing async client will be merged.